src/libs/SubscriptionUtils.ts

```diff
@@ -270,6 +270,7 @@ function getSubscriptionStatus(): SubscriptionStatus | undefined {
             // 2. Policy owner with amount owed, overdue (past grace period)
             if (hasGracePeriodOverdue()) {
                 return {
+                    isError: true,
```

Comment:
Can you also check if there's another status that `isError` should be true as well?

Reply:
Little bump

Reply:
Yes, they should be the same.
We also have this function:
```javascript
function hasSubscriptionGreenDotInfo(): boolean {
    return getSubscriptionStatus()?.isError === false;
}
```

So I think we just need to make sure, if `isError` doesn't equal true, leave it undefined.

Reply:
little bump, let's update it.

Reply:
Thanks. Please update the tests as well